### PR TITLE
[mass Obsolete] children of urinary system disorder

### LINKED ISSUE #6696: [mass Obsolete] children of urinary system disorder
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0100343 | antenatal Bartter syndrome
-- | -- | -- | --
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0019723 | disease of glomerular basement membrane
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0016579 | dominant hypophosphatemia with nephrolithiasis or osteoporosis
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0016352 | idiopathic inherited hypercalciuria
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0017891 | inherited renal cancer-predisposing syndrome
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0019743 | nephropathy secondary to a storage or other metabolic disease
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0022409 | nephropathy-associated ciliopathy
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0019720 | non-syndromic renal or urinary tract malformation
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0017828 | primary renal tubular acidosis
http://purl.obolibrary.org/obo/MONDO_0002118 | urinary system disorder | http://purl.obolibrary.org/obo/MONDO_0019724 | secondary glomerular disease

MONDO:0100343
MONDO:0019723
MONDO:0016579
MONDO:0016352
MONDO:0017891
MONDO:0019743
MONDO:0022409
MONDO:0019720
MONDO:0017828
MONDO:0019724

[MONDO:0100343](http://purl.obolibrary.org/obo/MONDO_0100343)
[MONDO:0019723](http://purl.obolibrary.org/obo/MONDO_0019723)
[MONDO:0016579](http://purl.obolibrary.org/obo/MONDO_0016579)
[MONDO:0016352](http://purl.obolibrary.org/obo/MONDO_0016352)
[MONDO:0017891](http://purl.obolibrary.org/obo/MONDO_0017891)
[MONDO:0019743](http://purl.obolibrary.org/obo/MONDO_0019743)
[MONDO:0022409](http://purl.obolibrary.org/obo/MONDO_0022409)
[MONDO:0019720](http://purl.obolibrary.org/obo/MONDO_0019720)
[MONDO:0017828](http://purl.obolibrary.org/obo/MONDO_0017828)
[MONDO:0019724](http://purl.obolibrary.org/obo/MONDO_0019724)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6696

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -453438,18 +453367,17 @@ property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-520
...
-comment: Reason of obsoletion: out of scope - MONDO:excludeGroupingClass. This is part of a large mass-obsoletione effort for ORDO terms.
-subset: obsoletion_candidate
+name: obsolete antenatal Bartter syndrome
+def: "OBSOLETE. A phenotypic variant of Bartter syndrome presenting antenatally with maternal polyhydramnios, pre-term delivery and postnatally with polyuria, and nephrocalcinosis. Hypokalemic alkalosis, increased levels of plasma renin and aldosterone, low blood pressure and vascular resistance to angiotensin II are characteristically associated. Genotypically they comprise Type 1 and Type 2 Bartter syndrome." [Orphanet:93604]
 synonym: "Bartter syndrome, furosemide type" EXACT [Orphanet:93604]
 synonym: "Bartter syndrome, furosemide-amiloride type" EXACT [Orphanet:93604]
 synonym: "hyperprostaglandin E syndrome" EXACT [Orphanet:93604]
-xref: Orphanet:93604 {source="MONDO:equivalentObsolete"}
-is_a: MONDO:0015231 {source="Orphanet:93604", source="https://github.com/monarch-initiative/mondo/issues/2935", source="https://orcid.org/0000-0001-5493-2602"} ! Bartter syndrome
+xref: Orphanet:93604 {source="MONDO:obsoleteEquivalentObsolete"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:453375]: @nicolevasilevsky I thought this was `{source="MONDO:obsoleteEquivalent"}` - did this change?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -453438,18 +453367,17 @@ property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-520
...
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeGrouping"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:453377]: @nicolevasilevsky this used to be `property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeGroupingClass"}` did this change?

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -377826,36 +377775,25 @@ relationship: has_characteristic MONDO:0021136 {source="MONDO:0019750"} ! rare
...
 synonym: "basement membrane disease" BROAD [Orphanet:93550]
-xref: Orphanet:93550 {source="MONDO:equivalentObsolete"}
-xref: UMLS:CN206630 {source="MONDO:equivalentTo"}
-is_a: MONDO:0005240 {source="Orphanet:93550", source="Orphanet:93550/inferred"} ! kidney disorder
-is_a: MONDO:0019722 {source="MONDO:Redundant", source="Orphanet:93550"} ! glomerular disorder
-intersection_of: MONDO:0000001 ! disease
-intersection_of: disease_has_location GO:0005604 ! basement membrane
-intersection_of: disease_has_location UBERON:0005777 ! glomerular basement membrane
-relationship: has_characteristic MONDO:0021136 {source="MONDO:0020030"} ! rare
+xref: Orphanet:93550 {source="MONDO:obsoleteEquivalentObsolete"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:377780]: should this be `xref: Orphanet:93550 {source="MONDO:obsoleteEquivalent"}`?

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -346396,7 +346357,6 @@ xref: ICD10CM:I42.2 {source="Orphanet:324525/attributed", source="Orphanet:32452
 xref: Orphanet:324525 {source="MONDO:equivalentTo"}
 xref: UMLS:CN204073 {source="MONDO:equivalentTo"}
 is_a: MONDO:0016387 ! mitochondrial oxidative phosphorylation disorder
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:346359]: Flagging since this is the only remaining `is_a` and it does not have a properly formatted source.

**Review by twhetzel — CHANGES_REQUESTED** [review]
Comments inline

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -321658,20 +321638,15 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/730/anti-hla
 
 [Term]
 id: MONDO:0016352
-name: idiopathic inherited hypercalciuria
-comment: Reason of obsoletion: out of scope - MONDO:excludeGroupingClass. This is part of a large mass-obsoletione effort for ORDO terms.
-subset: obsoletion_candidate
-subset: ordo_disease {source="Orphanet:2197"}
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:321664]: yes, we're removing those now. please update your checklist

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:346359]: I added my orcid

**Pull request merged**